frontend/src/concepts/hardwareProfiles/utils.ts

```diff
@@ -37,7 +37,10 @@ export const formatNodeSelector = (selector: NodeSelector): string[] =>
   Object.entries(selector).map(([key, value]) => `Key = ${key}; Value = ${value}`);
 
 export const formatResource = (identifier: string, request: string, limit: string): string =>
-  `${identifier}: Request = ${request}; Limit = ${limit}`;
+  `${identifier}: Request = ${formatResourceValue(
+    request,
+    identifier,
+  )}; Limit = ${formatResourceValue(limit, identifier)}`;
```

Comment:
we need to use the resource type which comes from the hardware profile. technically the identifier could be something like: `kingston.com/experimental-memory` but still use memory units

Reply:
Can you rephrase? I didn't understand your suggestion.

Reply:
I have already modified `formatResourceValue` to take string as input, because the data coming from Hardware profile is string

Reply:
I am thinking about this

```tsx
export const formatResource = (identifier: string, request: string, limit: string, resourceType: IdentifierResourceType): string =>
  `${identifier}: Request = ${formatResourceValue(
    request,
    resourceType,
  )}; Limit = ${formatResourceValue(
    limit,
    resourceType,
  )}`;
```